Pi_Code/Submarine.py

```diff
@@ -67,7 +75,30 @@ def UpdateDepth(self):
 				self._messageBoard.LogMessage(message)
 			else:
 				self._depth = message
-	
+
+	def UpdateJoystick(self):
+		# Check to see if Joystick sent a message
+		if(self.controllerConn.poll()):
+			message = self.controllerConn.recv()
+			if (message):
+				if isinstance(message, tuple):
+					# This means we have motor controls
+					# Speed and bool:Is Forward
+					self._messageBoard.SendLeftSpeedPacket(message[0],(message[0]>=0))
```

Comment:
No this is not how the fucntion is to be used ...

Reply:
Not sure I understand the issue entirely. We send a message to the messageBoard with two values, an integer 0-100 and a boolean, where if it is True then the motor goes backwards. That is how I understood the messageBoard calls. 

I just changed it to `abs(message[n])` so that it is always a positive number sent to the message board, and I get the boolean direction by checking if the throttle position says it is less than zero. 

If that's not correct, should we discuss it later today?